### ISSUE TITLE
bug: In extras all the high contrast themese are the same as the default one

### ISSUE THREAD

**Issue by lespea** [pauchiner/pastelnight.nvim#23]
### Did you check docs and existing issues?

- [X] I have read all the pastelnight.nvim docs
- [X] I have searched the existing issues of pastelnight.nvim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

n/a

### Operating system/version

n/a

### Describe the bug

Went to try out the new high contrast themes and in all the places I checked there was no difference (fish, fish_themes, and kitty).  I assume it is some bug in the generator or maybe it's intentional for the extras?

### Steps To Reproduce

n/a

### Expected Behavior

n/a

**Comment by pauchiner** [comment]
You are right, it was a mistake in the code that generates the extras, I just merge a PR with the problem solved. 

Thanks for the issue 😁.

**Comment by pauchiner** [comment]
Also, i renamed the high contrast variant from `pastelnight-high-contrast` to `pastelnight-highcontrast`
